src/main/java/nextstep/member/domain/Member.java

```diff
@@ -22,6 +22,21 @@ public Member(String email, String password, Integer age) {
         this.age = age;
     }
 
+    public Member(Long id, String email, String password, Integer age) {
+        this.id = id;
+        this.email = email;
+        this.password = password;
+        this.age = age;
+    }
+
+    public static Member of(String email, String password, Integer age) {
+        return new Member(email, password, age);
```

Comment:
다른 정적 팩토리 메서드를 호출하는 방식으로 변경해보는 건 어떨까요?
```suggestion
        return of(null, email, password, age);
```